data/rdm/manufacturer_names.proto

```diff
@@ -630,10 +1066,22 @@ manufacturer {
   manufacturer_id: 20781
   manufacturer_name: "DJPOWER ELECTRONIC STAGE LIGHTING FIXTURE FACTORY (GUANGZHOU)"
 }
+manufacturer {
+  manufacturer_id: 434
+  manufacturer_name: "DJSI Schinstad ANS (Northern Light)"
```

Comment:
should we codespell-ignore those somehow?

Reply:
Yeah I forgot to revisit that, now done and will hopefully come out clean...